helm/_mailpit/values.yaml

```diff
@@ -73,3 +77,13 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+mailpit:
```

Comment:
1. this helm chart is for `mailpit`, so having `mailpit` attribute does not makes sense.
2. as mount volume path is hard coded, just hard code the path with `args` as well.